scripts/publish-app.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+var ghpages = require('gh-pages');
+var path = require('path');
+var fs = require('fs');
+var args = require('yargs')
+  .boolean('p')
+  .default('p', false)
+  .alias('p', 'push')
+  .describe('p', 'Push changes to remote repository, i.e. do an actual publish')
+  .alias('v', 'version')
+  .describe('v', 'semver version to publish the app as')
+  .demandOption(['p'])
+  .demandCommand(1)
+  .usage('Usage: $0 <app name> -p ')
+  .argv;
+
+// Debug tooling
+process.env.NODE_DEBUG = 'gh-pages';
+
+// get version from lerna global version
+var version = args.version || require(path.resolve(__dirname, '../lerna.json')).version;
+var appName = args._[0];
+
+var appInfo = {
+  server: {
```

Comment:
[Suggestion] It will be cool to have this as separate json later. 
Nothing really needed. Just suggestion.